src/js/components/Provenance/Tables/DistributionsTable.tsx

```diff
@@ -58,7 +63,9 @@ const DistributionsTable = ({ distributions }: DistributionsTableProps) => {
     [td]
   );
 
-  return <BaseProvenanceTable dataSource={distributions} columns={columns} />;
+  return (
+    <BaseProvenanceTable dataSource={distributions} columns={columns} rowKey={(record) => record.access.landingPage} />
```

Comment:
in the types `landingPage` is required, but is it the case that it's occasionally missing in production instances?

Reply:
This was deliberate, since landing page is a required part of "access"